doc/nrf/known_issues.rst

```diff
@@ -387,6 +387,14 @@ NCSDK-6238: Socket API calls may hang when using Download client
 
 **Workaround:** Set :option:`CONFIG_DOWNLOAD_CLIENT_RANGE_REQUESTS`.
 
+NCSDK-6856: File content may be wrong when using :ref:`lib_download_client` with conten range requests.
+  When using :ref:`lib_download_client` with range requests (required for TLS downloads, optionally otherwise), the HTTP response parsing may fail and cause corrupt file download.
```

Comment:
```suggestion
  When using :ref:`lib_download_client` with range requests (required for TLS downloads, optionally otherwise), the HTTP response parsing may fail and cause a corrupted file download.
```